lib/player.js

```diff
@@ -1855,7 +1860,13 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
 
     // If start time is not provided and ServiceDescription exists
     // Offset start time by ServiceDescription.latency.target
-    if (!has.startTime && this.parser_ instanceof shaka.dash.DashParser) {
+    if (this.isLive() &&
+      wants.startTimeOffsetMs !== undefined &&
+      wants.startTimeOffsetMs !== null) {
+      has.startTime = this.manifest_.presentationTimeline
+          .getSegmentAvailabilityEnd() - wants.startTimeOffsetMs;
```

Comment:
can we use this.config_.streaming.liveCatchUp.targetLiveLatencyOverride instead of startTimeOffsetMs?

Reply:
Yes, forgot to modify this section. Will revert all modifications on payload